app/views/issues/_attributes.rhtml

```diff
@@ -1091,6 +1101,11 @@ Event.observe(window, 'load', function() {
         }
     }
 
+    var partsStatus = partsStatusInput.value
+    if (statusInput.value == 5 && partsStatus == 'Pending') {
```

Comment:
```suggestion
    if (statusInput.value == 5 && !partsStatus) {
```
Please test the above suggestion
use the same logic above (line 1094) for closed statuses